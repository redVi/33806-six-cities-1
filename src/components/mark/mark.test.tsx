import React from "react";
import renderer from "react-test-renderer";
import Mark from "@/components/mark/mark";

describe("Mark", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Mark />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
