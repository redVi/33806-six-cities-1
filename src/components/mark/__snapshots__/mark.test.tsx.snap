// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mark renders correctly 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;
